shared/utils/__tests__/logic.spec.ts

```diff
@@ -54,13 +48,13 @@ describe('Logic validation', () => {
     const LOGIC_RESPONSE = makeResponse(LOGIC_FIELD._id, 'lorem')
     const MOCK_LOGIC_ID = new ObjectId().toHexString()
 
-    let form: IFormDocument
+    let form: PickLogicSubset
 
     beforeEach(() => {
       form = {
         _id: new ObjectId(),
         form_fields: [CONDITION_FIELD, LOGIC_FIELD],
-      } as IFormDocument
+      } as unknown as PickLogicSubset
```

Comment:
I think the types in `/shared` are strong enough to not have to cast as unknown. 
The reason why you still need to cast as unknown for now is due to using `ObjectId` which is incompatible with `PickLogicSubset['_id']` type. (just use a string for `_id`)